gnovm/cmd/gno/test.go

```diff
@@ -123,6 +127,13 @@ func (c *testCfg) RegisterFlags(fs *flag.FlagSet) {
 		"test name filtering pattern",
 	)
 
+	fs.IntVar(
+		&c.parallel,
+		"parallel",
+		1,
+		"number of package to run concurrently (default(1): sequentially); 0 = GOMAXPROCESS",
```

Comment:
It could be great to run tests in parallel within the same package as well. A recent change I made was to ensure each test is independent; if there is a global variable, it is always set up again. This is probably easy to implement.